xml/System.IO/Directory.xml

```diff
@@ -3900,7 +3900,7 @@ Directory::CreateDirectory("Public\\Html");
       </Parameters>
       <Docs>
         <param name="path">The file or directory for which to set the creation date and time information.</param>
-        <param name="creationTime">The date and time the file or directory was last written to. This value is expressed in local time.</param>
+        <param f>The date and time the file or directory was last written to. This value is expressed in local time.</param>
```

Comment:
Param names are auto-generated from assemblies so this part shouldn't be changed.

```suggestion
        <param name="creationTime">The date and time the file or directory was last written to. This value is expressed in local time.</param>
```